pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -22,14 +22,20 @@ import (
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type PVCToMigrate struct {
+	Name               string                            `json:"name,omitempty"`
+	Namespace          string                            `json:"namespace,omitempty"`
```

Comment:
Do we need a TargetNamespace here too, in case the MigPlan has a namespace mapping, or are there other aspects here which are known not to support that yet which will need to be fixed before namespace mappings work?

Reply:
Good catch. There is nothing technically that is stopping namespace mappings from working, we just need to tweak the code a bit. I will add support for that here.